### PR TITLE
Deploy cparser-builder after all

### PR DESCRIPTION
**Pull request by lsf37**
This mostly reverts commit 467e79db9a49efda2ce2296a12ffaa48c8af0720 and deploys the cparser image after all, because it's too hard to figure out how to refer to a locally built image in the GitHub docker action.

### CODE REVIEW

**Comment by lsf37** [comment]
(See also discussion on #106)

**Comment by lsf37** [comment]
@mbrcknl is this what you had in mind?

File: .github/workflows/deploy-cparser-builder.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  push:
+    branches:
+    - master
+    paths:
+    - 'scripts/**'
+    - 'docker/**'
+    # trigger on anything the follow-on deployments might need as well:
+    - 'seL4-platforms/**'
+    - 'cparser-run/**'
+    - 'preprocess/**'
```
Comment (mbrcknl) [inline .github/workflows/deploy-cparser-builder.yml:25]: It might have been ok to leave these triggers in the downstream workflows, but this is fine.

**Review by mbrcknl — APPROVED** [review]
Yep, this is pretty much what I was imagining.

**Pull request merged**